src/sentry/models/environmentbookmark.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import BaseManager, FlexibleForeignKey, Model, sane_repr
+from sentry.models import Environment
+
+
+class EnvironmentBookmark(Model):
```

Comment:
@markstory what do I need to do to support having this be deleted when organizations are deleted?

Suppose I probably need to support deletion for when the environment itself is deleted. I forget if we have some cascade thing cc @wedamija

Reply:
This will cascade delete automatically when envs are deleted as part of org deletion. If there could potentially be a lot of rows, it might make sense to add a specific deletion for objects related to environments here: https://github.com/getsentry/sentry/tree/9ced05b0cf494b8398b5ce1c23957b620a8619ee/src/sentry/deletions/defaults

Reply:
Where is the cascade defined? Is that by default?

Reply:
Yeah, cascade used to be the default in django, and we added it in when we upgraded here: https://github.com/getsentry/sentry/blob/22758802ea4cea64e317d683977298c448fed145/src/sentry/db/models/fields/foreignkey.py#L14